nrfx_config_nrf9160.h

```diff
@@ -36,243 +36,83 @@
-#define NRF_EGU0 \
-        ((NRF_EGU_Type *)DT_NORDIC_NRF_EGU_EGU_0_BASE_ADDRESS)
-#endif
-#ifdef DT_NORDIC_NRF_EGU_EGU_1_BASE_ADDRESS
-#define NRF_EGU1 \
-        ((NRF_EGU_Type *)DT_NORDIC_NRF_EGU_EGU_1_BASE_ADDRESS)
-#endif
-#ifdef DT_NORDIC_NRF_EGU_EGU_2_BASE_ADDRESS
-#define NRF_EGU2 \
-        ((NRF_EGU_Type *)DT_NORDIC_NRF_EGU_EGU_2_BASE_ADDRESS)
-#endif
-#ifdef DT_NORDIC_NRF_EGU_EGU_3_BASE_ADDRESS
-#define NRF_EGU3 \
-        ((NRF_EGU_Type *)DT_NORDIC_NRF_EGU_EGU_3_BASE_ADDRESS)
-#endif
-#ifdef DT_NORDIC_NRF_EGU_EGU_4_BASE_ADDRESS
-#define NRF_EGU4 \
-        ((NRF_EGU_Type *)DT_NORDIC_NRF_EGU_EGU_4_BASE_ADDRESS)
-#endif
-#ifdef DT_NORDIC_NRF_EGU_EGU_5_BASE_ADDRESS
-#define NRF_EGU5 \
-        ((NRF_EGU_Type *)DT_NORDIC_NRF_EGU_EGU_5_BASE_ADDRESS)
-#endif
-
-#ifdef DT_INST_0_NORDIC_NRF_FICR_BASE_ADDRESS
-#define NRF_FICR \
-        ((NRF_FICR_Type *)DT_INST_0_NORDIC_NRF_FICR_BASE_ADDRESS)
-#endif
-
-#ifdef DT_NORDIC_NRF_GPIO_GPIO_0_BASE_ADDRESS
-#define NRF_P0 \
-        ((NRF_GPIO_Type *)DT_NORDIC_NRF_GPIO_GPIO_0_BASE_ADDRESS)
-#endif
-
-#ifdef DT_NORDIC_NRF_GPIOTE_GPIOTE_0_BASE_ADDRESS
-#define NRF_GPIOTE \
-        ((NRF_GPIOTE_Type *)DT_NORDIC_NRF_GPIOTE_GPIOTE_0_BASE_ADDRESS)
-#endif
-
-#ifdef DT_NORDIC_NRF_I2S_I2S_0_BASE_ADDRESS
-#define NRF_I2S \
-        ((NRF_I2S_Type *)DT_NORDIC_NRF_I2S_I2S_0_BASE_ADDRESS)
-#endif
-
-#ifdef DT_INST_0_NORDIC_NRF_KMU_BASE_ADDRESS
-#define NRF_KMU \
-        ((NRF_KMU_Type *)DT_INST_0_NORDIC_NRF_KMU_BASE_ADDRESS)
-#endif
-
-#ifdef DT_INST_0_NORDIC_NRF91_FLASH_CONTROLLER_BASE_ADDRESS
-#define NRF_NVMC \
-        ((NRF_NVMC_Type *)DT_INST_0_NORDIC_NRF91_FLASH_CONTROLLER_BASE_ADDRESS)
-#endif
-
-#ifdef DT_NORDIC_NRF_PDM_PDM_0_BASE_ADDRESS
-#define NRF_PDM \
-        ((NRF_PDM_Type *)DT_NORDIC_NRF_PDM_PDM_0_BASE_ADDRESS)
-#endif
-
-#ifdef DT_INST_0_NORDIC_NRF_POWER_BASE_ADDRESS
-#define NRF_POWER \
-        ((NRF_POWER_Type *)DT_INST_0_NORDIC_NRF_POWER_BASE_ADDRESS)
-#endif
-
-#ifdef DT_NORDIC_NRF_PWM_PWM_0_BASE_ADDRESS
-#define NRF_PWM0 \
-        ((NRF_PWM_Type *)DT_NORDIC_NRF_PWM_PWM_0_BASE_ADDRESS)
-#endif
-#ifdef DT_NORDIC_NRF_PWM_PWM_1_BASE_ADDRESS
-#define NRF_PWM1 \
-        ((NRF_PWM_Type *)DT_NORDIC_NRF_PWM_PWM_1_BASE_ADDRESS)
-#endif
-#ifdef DT_NORDIC_NRF_PWM_PWM_2_BASE_ADDRESS
-#define NRF_PWM2 \
-        ((NRF_PWM_Type *)DT_NORDIC_NRF_PWM_PWM_2_BASE_ADDRESS)
-#endif
-#ifdef DT_NORDIC_NRF_PWM_PWM_3_BASE_ADDRESS
-#define NRF_PWM3 \
-        ((NRF_PWM_Type *)DT_NORDIC_NRF_PWM_PWM_3_BASE_ADDRESS)
-#endif
-
-#ifdef DT_INST_0_NORDIC_NRF_REGULATORS_BASE_ADDRESS
-#define NRF_REGULATORS \
-        ((NRF_REGULATORS_Type *)DT_INST_0_NORDIC_NRF_REGULATORS_BASE_ADDRESS)
-#endif
-
-#ifdef DT_NORDIC_NRF_RTC_RTC_0_BASE_ADDRESS
-#define NRF_RTC0 \
-        ((NRF_RTC_Type *)DT_NORDIC_NRF_RTC_RTC_0_BASE_ADDRESS)
-#endif
-#ifdef DT_NORDIC_NRF_RTC_RTC_1_BASE_ADDRESS
-#define NRF_RTC1 \
-        ((NRF_RTC_Type *)DT_NORDIC_NRF_RTC_RTC_1_BASE_ADDRESS)
-#endif
+#if defined(CONFIG_ARM_NONSECURE_FIRMWARE)
```

Comment:
Also, I am wondering: why is this not becoming part of MDK? They already have a define for TRUSTZONE...